docs/Development-Tips.md

```diff
@@ -0,0 +1,29 @@
+Below is a collection of development tips when running CodaLab locally.
+
```

Comment:
For development, use Python3.6 is better. Other higher version of Python3 might brings problem when installing python packages.